docs/java-api/docs/bulk.asciidoc

```diff
@@ -165,3 +165,27 @@ client.admin().indices().prepareRefresh().get();
 client.prepareSearch().get();
 --------------------------------------------------
 
+
+[[java-docs-bulk-global-parameters]]
+==== Global Parameters
+
+Similarly to REST API where you can provide URL parameters, you can specify global parameters on BulkRequest as well
```

Comment:
Global parameters can be specified on the BulkRequest and well as BulkProcessor, similar to the REST API.